ontologies/Items.ttl

```diff
@@ -0,0 +1,171 @@
+	rdfs:label "Location"@en ;
+	rdfs:comment "The broad division of the Library or its storage areas within which an item is held: for instance, the History of Medicine collection, or the Closed Store Archives and Manuscripts material."@en ;
+ 	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+  
+wi:RequestStatus rdf:type owl:Class ;
+	rdfs:label "RequestStatus"@en ;
+	rdfs:comment "A note of whether the item in question is on open shelves in the Wellcome Library, or must be ordered from closed stacks, or is not orderable for reasons such as closure under Data Protection legislation."@en ;
+ 	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+	
+#######  Object properties ##### 
+  
+wi:hasIdentifier rdf:type owl:ObjectProperty ;
+	rdfs:label "hasIdentifier"@en ;
+	rdfs:comment "Relates the item to a unique system-generated identifier that governs interaction between systems and is regarded as canonical within the Wellcome data ecosystem."@en ; 
+	rdfs:domain wi:Item ;
+	rdfs:range wi:Identifier ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+
+wi:hasSourceIdentifier rdf:type owl:ObjectProperty ;
+	rdfs:label "hasSourceIdentifier"@en ;
+	rdfs:comment "Relates an item to an identifier used within one of the Wellcome Library's databases or other systems to refer to works or items within the Library's holdings, whether for internal technical purposes or for human citation."@
+	rdfs:domain wi:Item
+	rdfs:range wi:SourceIdentifier
+	rdfs:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+
+wi:hasSourceIdentifierAuthority rdf:type owl:ObjectProperty ;
+	rdfs:label "hasSourceIdentifierAuthority" @en ;
+	rdfs:comment "Relates a SourceIdentifier to a particular authoritative source of such identifiers: for example, if the SourceIdentifier is MS.49 this property might indicate that this identifier has its origins in the Wellcome Library's CALM archive management system."@en ;
+	rdfs:domain wi:SourceIdentifier
+	rdfs:range wi:SourceIdentifierAuthority
+	rdfs:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+
+wi:hasClassmark rdf:type owl:ObjectProperty ;
+	rdfs:label "hasClassmark"@en ;
+	rdfs:comment "This relates an item to a classmark, the latter being a code that relates a work to a classification of Library holdings, often but not always relating to a subject-based scheme of library classification and arrangement such as Barnard or the National Library of Medicine classification."@en ;
+	rdfs:domain wi:Item ;
+	rdfs:range wi:Classmark ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .	
+  
+wi:hasLocation rdf:type owl:ObjectProperty ;
+	rdfs:label "hasLocation"@en ;
+	rdfs:comment "This relates an item to a broad division of the Library or its storage areas within which an item is held: for instance, the History of Medicine collection, or the Closed Store Archives and Manuscripts material."@en ;
```

Comment:
Good point.  Intellectual division (though that may have physical implications, of course).